FoodBookApp/UI/Views/Browse/BrowseView.swift

```diff
@@ -20,7 +20,7 @@ struct BrowseView: View {
         ScrollView {
             Group {
                 ForEach(searchResults, id: \.self) { spot in
-                    NavigationLink(destination: SpotDetailView()){ // TODO: In the future this should have the SpotId as param
+                    NavigationLink(destination: SpotDetailView(spotId: "ms1hTTxzVkiJElZiYHAT")){ // TODO: @JuanDiego
```

Comment:
This is what currently makes all details to go to MiCaserito. @juanyepesp once you have the backend integratoin this should probably be like `spot.id`.